setup_wizard/character_rig_setup/rig_script.py

```diff
@@ -3109,3 +3132,29 @@ def setup_finger_scale_controls_on_x_axis_to_curl_just_the_fingertips(rigified_r
     for side in [".L", ".R"]:
         for bone in fingerlist:
             rigified_rig.pose.bones[bone + side].lock_scale[0] = False
+
+
+# Let's 'exclude' that wgt collection: https://blenderartists.org/t/disable-exlude-from-view-layer-in-collection/1324744
+def searchForLayerCollection(layerColl, coll_name):
+    found = None
+    if (layerColl.name == coll_name):
+        return layerColl
+    for layer in layerColl.children:
+        found = searchForLayerCollection(layer, coll_name)
+        if found:
+            return found

```

Comment:
Wanted to change this method a tiny bit so that we can provide a collection name as opposed to having a hardcoded collection name.

Reply:
lgtm